packages/patternfly-4/react-integration/cypress/integration/button.spec.ts

```diff
@@ -0,0 +1,31 @@
+describe('Button Demo Test', () => {
+  it('Navigate to demo section', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('#button-demo-nav-item-link').click();
+    cy.url().should('eq', 'http://localhost:3000/button-demo-nav-link')
+  });
+
+  it('Verify primary button', () => {
+    cy.get('button').should('have.class', 'pf-m-primary')
+  }); 
+
+  it('Verify primary button clicks', () => {
+    cy.get('.pf-m-primary').click()
+  })
+
+  it('Verify secondary button', () => {
+    cy.get('button').should('have.class', 'pf-m-secondary')
+  }); 
+
+  it('Verify secondary button clicks', () => {
+    cy.get('.pf-m-secondary').click()
```

Comment:
It would be nice to add some simple functionality to the button That way we are testing the on click event.

Reply:
Agree.  When creating a test should also make sure that you have a verification step.  This way if we break something with the functionality we will catch it.